Makefile

```diff
@@ -48,6 +48,11 @@ destroy-box:
 
 remove-tmp:
 	rm -rf ./tmp
+    rm -rf ./.vagrant
+    rm -rf ./.minio.sys
+    rm -rf ./example/.terraform
+    rm -rf ./example/terraform.tfstate
```

Comment:
Formatting 
![image](https://user-images.githubusercontent.com/51820995/93583959-ea5ae680-f9a4-11ea-8307-e89530bf1b65.png)

Let me know when it's fixed, then I'll approve :D

Reply:
It looked good on my end, but pushed another update now :)

Reply:
Sweet!